components/reddit/sources/new-links-by-user/new-links-by-user.js

```diff
@@ -0,0 +1,78 @@
+const common = require("../common");
+const { reddit } = common.props;
+
+module.exports = {
+  ...common,
+  key: "new-links-by-user",
+  name: "New links by user",
+  description: "Emits an event each time a user posts a new link.",
+  version: "0.0.1",
+  dedupe: "unique",
+  props: {
+    ...common.props,
+    username: { propDefinition: [common.props.reddit, "username"] },
+    numberOfParents: {
```

Comment:
Can you make this prop optional, as well?

Reply:
Yes, this was made optional as well. There were three instances, with different min/max range, so had to use a different default for each one.